packages/flame/lib/src/particles/particle.dart

```diff
@@ -13,18 +14,19 @@ import 'rotating_particle.dart';
 import 'scaled_particle.dart';
 import 'translated_particle.dart';
 
-/// A function which returns [Particle] when called
+/// A function which returns a [Particle] when called.
 typedef ParticleGenerator = Particle Function(int);
 
 /// Base class implementing common behavior for all the particles.
 ///
-/// Intention is to follow same "Extreme Composability" style
-/// as across the whole Flutter framework, so each type of particle implements
-/// some particular behavior which then could be nested and combined together
-/// to create specifically required experience.
+/// Intention is to follow the same "Extreme Composability" style as seen across
+/// the whole Flutter framework. Each type of particle implements some
+/// particular behavior which then could be nested and combined to create
+/// specifically required experience.
```

Comment:
Strange last part of the sentence?

Reply:
Suggestion for rewrite?

Reply:
"The experience you are looking for" or something like that maybe?